modules/distr-tracing-otel-install-web-console.adoc

```diff
@@ -0,0 +1,119 @@
+////
+This module included in the following assemblies:
+- distr_tracing_otel/distr-tracing-otel-installing.adoc
+////
+
+:_content-type: PROCEDURE
+[id="distr-tracing-install-otel-operator_{context}"]
+= Installing the {OTELShortName} by using the OpenShift web console
+
+You can install the {OTELShortName} by using the *Administrator* view of the OpenShift web console.
+
+.Prerequisites
+
+* You are logged in to the OpenShift web console as a cluster administrator with the `cluster-admin` role. For {product-dedicated}, you must be logged in using an account with the `dedicated-admin` role.
+
+* An active OpenShift CLI (`oc`) session by a cluster administrator with the `cluster-admin` role.
```

Comment:
Should make the second bullet a full sentence parallel with the first one.

Reply:
Thank you, I never thought of this!